docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -1,387 +1,390 @@
+
+## Get Started
+
+Follow these steps to install and build the micro-rdk on your ESP32:
+
+1. Install the [required software](#software-requirements)
+2. [Set up your development enviroment](#set-up-your-development-enviroment) with Viam's [Canon CLI utility](#use-viams-canon-cli-utility) *(recommended)* or [manually](#set-up-development-environment-manually)
+3. [Install the Micro-RDK](#install-the-micro-rdk)
+
+### Hardware Requirements
+
+You need an Espressif ESP32 microcontroller to use the micro-RDK.
+Viam recommends purchasing the ESP32 with a [development board](https://www.espressif.com/en/products/devkits).
+
+* **Minimal configuration:** 384kB Ram 4MB flash
+* **Recommended configuration:** 384kB Ram + 8MB SPIRAM + 4MB Flash
+
+### Software Requirements
+
+The micro-RDK is written in Rust.
+To be able to program the ESP32 on macOS and Linux systems, you must install the following software on your development machine:
+
+#### Install Dependencies
+
+{{< tabs >}}
+{{% tab name="Linux (Ubuntu)" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo apt-get install git libssl-dev dfu-util libusb-1.0-0 libudev-dev
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+
+If you haven't already, install Homebrew:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Then, install `dfu-util`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+brew install dfu-util
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+#### Install `Rust`
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+See the [Rust Installation guide](https://www.rust-lang.org/tools/install) for more information and other installation methods.
+
+#### Install `cargo-generate` with `cargo`
+
+ `cargo` installs automatically when downloading Rust with rustup.
+
+Run the following command to install `cargo-generate`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install cargo-generate
+```
+
+#### Install `espflash`
+
+Run the following command to install `espflash`
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install espflash
+```
+
+## Set up your development enviroment
+
+{{< tabs >}}
+{{% tab name="Canon CLI (recommended)" %}}
+
+### Use Viam's Canon CLI utility
+
+[Canon](https://github.com/viamrobotics/canon) is a CLI utility for managing a Docker-based canonical environment.
+
+Canon requires a working installation of Docker Engine.
+
+[Install Docker Engine](https://docs.docker.com/engine/install/) before proceeding.
+If you are running Docker Engine on Linux, make sure that you go through the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to run Docker automatically on startup.
+
+{{< tabs >}}
+{{% tab name="Linux (Ubuntu)" %}}
```

Comment:
Nothing is specific to "ubuntu" for canon. It works equally on all linux distros.

Reply:
done!